java/http/vertx/java-http-vertx-producer/src/main/java/HttpKafkaProducer.java

```diff
@@ -138,7 +107,6 @@ private Promise<List<OffsetRecordSent>> send(String topic, Span span) {
                     } else {
                         fut.fail(ar.cause());
                     }
-
```

Comment:
No need to delete the empty line? Having it between two different if conditions helps with the code readability => makes it more clear this is  separate `if` and not just another `else-if`.

Reply:
Line added back in